weather_dl_v2/cli/app/subcommands/download.py

```diff
@@ -75,8 +75,17 @@ def submit_download(
             help="Force redownload of partitions that were previously downloaded.",
         ),
     ] = False,
+    priority: Annotated[
+        int | None,
+        typer.Option(
+            "-p",
+            "--priority",
+            help="Set the priority for submitted config in ALL licenses. If not added, the config is added" \
+                "at the end of the queue.",
```

Comment:
nit: let's mention '0' being the highest priority.

Reply:
done